content/en/developers/dogstatsd/unix_socket.md

```diff
@@ -142,7 +151,7 @@ To enable the Agent DogStatsD UDS:
 To send metrics from shell scripts, or to test that DogStatsD is listening on the socket, use `netcat`. Most implementations of `netcat`, such as `netcat-openbsd` on Debian or `nmap-ncat` on RHEL, support Unix Socket traffic with the `-U` flag:
 
 ```shell
-echo -n "custom.metric.name:1|c" | nc -U -u -w1 /var/run/datadog/dsd.socket
+d
```

Comment:
```suggestion
d
```
Is this correct here that you are replacing the "echo -n" command just with "d" ?

Reply:
Oh weird, this is definitely a weird typo 😅